compat/src/index.js

```diff
@@ -207,7 +207,7 @@ function applyEventNormalization({ type, props }) {
 	let newProps = {};
 
 	for (let i in props) {
-		if (/^on(Ani|Tra)/.test(i)) {
+		if (/^on(Ani|Tra|Tou)/.test(i)) {
```

Comment:
#2124 might make this difficult to merge... sorry! FYI, this change should now be made in `events.js` I think